server/jetstream_cluster.go

```diff
@@ -1728,7 +1728,7 @@ func (js *jetStream) applyMetaEntries(entries []*Entry, ru *recoveryUpdates) (bo
 					js.processUpdateStreamAssignment(sa)
 				}
 			default:
-				panic("JetStream Cluster Unknown meta entry op type")
+				panic(fmt.Sprintf("JetStream Cluster Unknown meta entry op type! %v", entryOp(buf[0])))
```

Comment:
```suggestion
				panic(fmt.Sprintf("JetStream Cluster Unknown meta entry op type: %v!", entryOp(buf[0])))
```

Reply:
Should we log the more detailed stuff then panic with a simple unknown opcode?

This will not be a bug, fyi, this will be a fault, data corruption, etc. Hence the panic.